cmd/openshift-install/agent.go

```diff
@@ -44,22 +42,10 @@ var (
 			Use:   "image",
 			Short: "Generates a bootable image containing all the information needed to deploy a cluster",
 			Args:  cobra.ExactArgs(0),
-			PostRun: func(cmd *cobra.Command, args []string) {
-				cleanup := setupFileHook(rootOpts.dir)
```

Comment:
I'm not fully sure we could easily throw away this part as part of the command execution flow